src/components/ViewOptions/ViewOptions.jsx

```diff
@@ -1,8 +1,8 @@
 import "./ViewOptions.scss";
 import { useState } from "react";
 
-const ViewOptions = () => {
```

Comment:
Line 4: Please pass in a function as a prop so we can change routes from container.